data/json/items/book/driving.json

```
@@ -25,6 +54,78 @@
     "category": "manuals",
     "name": { "str": "Top Gear magazine" },
     "description": "Lots of articles about cars and driving techniques.",
+    "variants": [
+      {
+        "id": "mag_cars",
+        "name": { "str": "Top Gear", "str_pl": "issues of Top Gear" },
+        "description": "Lots of articles about cars and driving techniques.",
+        "weight": 1
+      },
+      {
+        "id": "mag_cars_1",
+        "name": { "str": "Automobile", "str_pl": "issues of Automobile" },
+        "description": "Nice looking magazine about automobile industry, with shiny images and long articles.  Dated 2019 year.",
+        "weight": 1
+      },
+      {
+        "id": "mag_cars_2",
+        "name": { "str": "Automotive Industries", "str_pl": "issues of Automotive Industries" },
+        "description": "Monthly annual about recent changes in transport industry.",
+        "weight": 1
+      },
+      {
+        "id": "mag_cars_3",
+        "name": { "str": "Autoweek", "str_pl": "issues of Autoweek" },
+        "description": "Shabby issue about new and old car reviews, motorsport and DIY.",
+        "weight": 1
+      },
+      {
+        "id": "mag_cars_4",
+        "name": { "str": "Car and Driver", "str_pl": "issues of Car and Driver" },
+        "description": "Absolutely generic annual about automotive industry.",
+        "fun": 0,
+        "weight": 1
+      },
+      {
+        "id": "mag_cars_5",
+        "name": { "str": "Car Craft", "str_pl": "issues of Car Craft" },
+        "description": "Flashy journal about cars, racing and mechanics.",
+        "weight": 1
+      },
+      {
+        "id": "mag_cars_6",
+        "name": { "str": "Diesel Power", "str_pl": "issues of Diesel Power" },
+        "description": "Automotive magazine, focused on heavily modifying trucks, SUVs, and cars powered by diesel.",
+        "weight": 1
+      },
+      {
+        "id": "mag_cars_7",
+        "name": { "str": "Cruisin' Style Magazine", "str_pl": "issues of Cruisin' Style Magazine" },
+        "description": "Really old issue about classic car maintenance and restoration.",
+        "weight": 1
+      },
+      {
+        "id": "mag_cars_8",
+        "name": { "str": "Hot Rod", "str_pl": "issues of Hot Rod" },
+        "description": "Monthly magazine about hot rodding, drag racing and muscle cars.",
+        "weight": 1
+      },
+      {
+        "id": "mag_cars_9",
+        "name": { "str": "Motor Trend", "str_pl": "issues of Motor Trend" },
+        "description": "Boring automobile magazine with big shiny label.",
+        "fun": -1,
+        "weight": 1
+      },
+      {
+        "id": "mag_cars_10",
+        "name": { "str": "Quatro Rodas", "str_pl": "issues of Quatro Rodas" },
+        "description": "It seems to be a car magazine, but you can undertand only images, as the text is wrote on another language.",
```

```suggestion
        "description": "It seems to be a car magazine, but you can only understand the images, as the text is written in another language.",
```

~~Take that bilinguals~~